### PR TITLE
AUT-3909: remove lambda provisioned from dev account

### PR DESCRIPTION
**Pull request by pskushwaha1**
## What
[AUT-3909]
remove lambda provisioned from dev account 
we are incurring total **USD 1,664.66** monthly for lambda  , which includes only concurrency cost in DEV account ( di-auth-development ) , concurrency is not required in non performance dev account 

## How to review

1. Code Review
2. Deploy to sandpit with `./deploy-authdevs -p`

[AUT-3909]: https://govukverify.atlassian.net/browse/AUT-3909?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: ci/terraform/account-management/.terraform.lock.hcl [hunk 1]
```diff
@@ -52,6 +52,25 @@ provider "registry.terraform.io/hashicorp/local" {
   ]
 }
 
+provider "registry.terraform.io/hashicorp/null" {
```
Comment (whi-tw) [inline ci/terraform/account-management/.terraform.lock.hcl:55]: This seems incorrect, could you please drop this change?

File: ci/terraform/test-services/.terraform.lock.hcl [hunk 2]
```diff
@@ -52,6 +52,25 @@ provider "registry.terraform.io/hashicorp/local" {
   ]
 }
 
+provider "registry.terraform.io/hashicorp/null" {
```
Comment (whi-tw) [inline ci/terraform/test-services/.terraform.lock.hcl:55]: as above:

> This seems incorrect, could you please drop this change?

**Review by whi-tw — CHANGES_REQUESTED** [review]
Other than the comments in the `.lock.hcl` files, this all looks good!

File: ci/terraform/test-services/.terraform.lock.hcl (hunk 2 above)
Comment (pskushwaha1) [reply ci/terraform/test-services/.terraform.lock.hcl]: done removed the HCL changes

**Review by whi-tw — APPROVED** [review]
LGTM

**Pull request merged**